superset/utils/pandas_postprocessing.py

```diff
@@ -275,6 +285,12 @@ def pivot(  # pylint: disable=too-many-arguments
         margins_name=marginal_distribution_name,
     )
 
+    if not drop_missing_columns and len(series_set) > 0 and not df.empty:
+        for col in df.columns:
+            series = "".join([str(_) for _ in col])
```

Comment:
Since we're reusing this `"".join`, could we make a lambda for it to avoid having the same logic twice?

Reply:
nice catch!